scanpy/preprocessing/_pca.py

```diff
@@ -126,7 +128,10 @@ def pca(
     if data_is_AnnData:
         adata = data.copy() if copy else data
     else:
-        adata = AnnData(data, dtype=data.dtype)
+        if pkg_version('anndata') < version.parse('0.8.0rc1'):
+            adata = AnnData(data, dtype=data.dtype)
+        else:
+            adata = AnnData(data)
```

Comment:
Not related right? I think fine, but would also be fine to make a small separate PR and just merge it.

Reply:
right